learn/wavemaker-release-notes/v10-7-0.md

```diff
@@ -65,6 +68,14 @@ Introduced the following methods to get default options provided for the Richtex
 - `getLib()`: Returns the library used for the widget.
 - `overrideDefaults(options)`: Customize the options which the library supports.
 
+### Backend Enhancements
+
+- For Rest/Swagger services, the rest call goes from the backend when you enable proxy, making the direct path empty in service definitions.
+- Introduced the Allowed host property in the security tab of the profile configuration. It prevents host header injections and lets application developers control the allowed hosts of the application.
+- Added support in the Deployed Apps portal to enable **Creation** and **Deployment** of application in 1.18 and 1.19 Kubernetes clusters in Digital Ocean.
+- Introduced the **Max Authentication Age** property in the project security configuration. It permits developers to allow users to stay authenticated for the specified duration when the identity provider (idP) allows.
+- Introduced the `Max Sessions` property under the "Login configuration" section. With this, a developer can configure the duration for a logged-in user's session's life.
```

Comment:
Incorrect. Configure number of concurrent sessions your application users can have by setting the property `Max Sessions`. This settings can be found in the Login Configuration tab of the Security menu.